### PR TITLE
#71: Fixed the validation checks and also cleaned up some code

### PR DESCRIPTION
**Pull request by vbadkar**
Steps to test:
* Check all the forms on the site(Registration, login, article, block, menu, etc).
* Try submitting entire form empty
* Try submitting the form with an empty field

Expected behaviour
* The user should get redirected to error page with appropriate message for the user

### CODE REVIEW

**Review by vbadkar — COMMENTED** [review]
Added images by mistake

File: custom_cms.sql [hunk 1]
```diff
@@ -3,7 +3,11 @@
 -- https://www.phpmyadmin.net/
 --
 -- Host: localhost:3306
+<<<<<<< HEAD
```
Comment (Libbna) [inline custom_cms.sql:6]: can you remove these arrows, it is from conflicts I guess. @vbadkar

**Review by Libbna — CHANGES_REQUESTED** [review]

File: custom_cms.sql (hunk 1 above)
Comment (vbadkar) [reply custom_cms.sql:6]: oh, okay i'll remove it.

Comment (vbadkar) [reply custom_cms.sql]: @Libbna Resolved the conflct, can you check it now?

**Review by Libbna — APPROVED** [review]

**Review by RChaubey16 — APPROVED** [review]
Screenshot from 2022-01-14 16-12-40

Working Fine !

**Pull request merged**